runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -766,11 +767,13 @@ impl<'a> VMLogic<'a> {
     pub fn input(&mut self, register_id: u64) -> Result<()> {
         self.result_state.gas_counter.pay_base(base)?;
 
-        self.registers.set(
+        let charge_bytes_gas = !self.config.deterministic_account_ids;
```

Comment:
I'm reusing the `deterministic_account_ids` feature, as I assume we want to bundle it together with that.
Or should I create a separate wasm config flag? @nagisa